src/controllers/blocks/BlocksController.ts

```diff
@@ -13,6 +13,15 @@ import AbstractController from '../AbstractController';
  * - (Optional) `number`: Block hash or height at which to query. If not provided, queries
  *   finalized head.
  *
+ * Query:
+ * - (Optional) `eventDocs`: When set to `true`, every event will have an extra
+ * 	`docs` property containing an array of strings that represent each line of
+ * 	events documentation
+ * - (Optional) `extrinsicDocs`: When set to `true`, every extrinsic will have an extra
+ * 	`docs` property containing an array of strings that represent each line of
```

Comment:
I think it's unfortunate that the docs come back as an array of strings rather than a single string (with newlines). Is that something that is easy to change?

Reply:
Yeah I guess we could do some parsing.. The array of string is the side effect of how the metadata is being generated.

Reply:
We certainly can do parsing, appending a new line char and then joining them with a string. I will make that change and update the sample response so we can look at it.